### PR TITLE
Add/remove wanted skills, personalities, horoscope, leaders; save/load; calculate numPrettyGirls

### PR DESCRIPTION
**Pull request by hmhv1222**
- Added functions of add/remove wanted skills, personalities, horoscope, team leaders, and change team leaders

- Added function of calculating number of pretty girls

- Modified save/load function

### CODE REVIEW

**Comment by vivtony00** [comment]
please pull the master 
and 
send the pull request again.

File: app/js/team.js [hunk 1]
```diff
@@ -1,60 +1,79 @@
...
+		"wantedHoroscopes": []
 	};
 		
-	
 
-	refPath =  eventName + "/admin";
-	retrieveOnceFirebase(firebase, refPath, function(data) {	
-
-		if ( data.child("param").val() != null ) {
+	refPath = "events/" + eventName + "/admin";
```
Comment (vivtony00) [inline app/js/team.js:51]: 如果轉event既path既話就全世界都要轉(eg.做緊admin.js既人)
下次可以講聲

File: app/js/team.js [hunk 2]
```diff
@@ -1,60 +1,79 @@
...
 		$('#text_event_name').text("Event name: " + eventName);
-		
 	}
-
 });
 
 angular.module('teamform-team-app', ['firebase'])
-.controller('TeamCtrl', ['$scope', '$firebaseObject', '$firebaseArray', 
-    function($scope, $firebaseObject, $firebaseArray) {
+app.controller("MyAuthCtrl", ["$scope", "$firebaseAuth",
```
Comment (vivtony00) [inline app/js/team.js:13]: 呢個唔係真係俾你copy&paste...
係inject個$firebaseAuth落個controller到...
13呢行改做
`.controller('TeamCtrl', ['$scope', '$firebaseObject', '$firebaseArray', "$firebaseAuth"`

File: app/js/team.js [hunk 3]
```diff
@@ -1,60 +1,79 @@
...
+  function($scope, $firebaseAuth)
```
Comment (vivtony00) [inline app/js/team.js:14]: 14呢行改做
`function($scope, $firebaseObject, $firebaseArray, $firebaseAuth)`

**Review by vivtony00 — CHANGES_REQUESTED** [review]
see the comment inside the code

File: app/js/team.js (hunk 2 above)
Comment (hmhv1222) [reply app/js/team.js:13]: ok this was done by wasin

Comment (hmhv1222) [inline app/js/team.js]: i didnt check

File: app/js/team.js (hunk 1 above)
Comment (hmhv1222) [reply app/js/team.js:51]: ok wont change anymore

**Comment by hmhv1222** [comment]
Corrected Wasin’s controller injection mistake

**Pull request closed without merging**